.github/workflows/ci-core.yml

```diff
@@ -398,12 +405,32 @@ jobs:
           SONAR_HOST_URL: ${{ secrets.SONAR_HOST_URL }}
           SONAR_SCANNER_OPTS: "-Xms6g -Xmx8g"
 
-  trigger-flaky-test-detection:
-    name: Find New Flaky Tests
+  trigger-flaky-test-detection-for-root-project:
```

Comment:
why not `trigger-flaky-test-detection-for-chainlink`? It's not a reusable workflow, so we can hardcode repo name

Reply:
@Tofel we trigger for "./go.mod" (main project that I call "root" here) as well as for "./deployment/go.mod". But, I can change it to "trigger-flaky-test-detection-for-chainlink-project" if you think it makes more sense.